### PR TITLE
GitHub Pages: clarify when user action is required

### LINKED ISSUE #13643: Custom Domain for GitHub Pages Docs: clarify when user needs to take action
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages

https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain-and-the-www-subdomain-variant

### What part(s) of the article would you like to see updated?

- https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages

  automatic securing of `www` is unclear. Leaves reader wondering "does that mean I don't have to do anything else?"

- https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain-and-the-www-subdomain-variant

  Docs say "To set up a www subdomain alongside the apex domain, you must first configure an apex domain, `which will create` an ALIAS, ANAME, or A record with your DNS provider". Which wrongly implies the new DNS record is a passive side effect rather than requiring a direct action.

  It would be clearer as "To set up a www subdomain alongside the apex domain, you must first configure an apex domain `by creating` an ALIAS, ANAME, or A record with your DNS provider"

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by alzafacon**
I was confused by the phrase "we automatically attempt to secure".
It helps to be explicit that user action is still required

### Why:

Closes https://github.com/github/docs/issues/13643

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@alzafacon 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by Fazsarah — COMMENTED** [review]
Approve

**Comment by janiceilene** [comment]
This has been approved by an SME, who would like us to:

> remove the second “but you need to make the DNS changes to use the `www` subdomain”

Now we're just waiting on a writer review! Thanks so much for your patience @alzafacon 💛

File: content/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ shortTitle: Custom domains in GitHub Pages
...
 
-We recommend always using a `www` subdomain, even if you also use an apex domain. When you create a new site with an apex domain, we automatically attempt to secure the `www` subdomain for use when serving your site's content. If you configure a `www` subdomain, we automatically attempt to secure the associated apex domain. For more information, see "[Managing a custom domain for your {% data variables.product.prodname_pages %} site](/articles/managing-a-custom-domain-for-your-github-pages-site)."
+We recommend always using a `www` subdomain, even if you also use an apex domain. When you create a new site with an apex domain, we automatically attempt to secure the `www` subdomain for use when serving your site's content, but you need to make the DNS changes to use the `www` subdomain. If you configure a `www` subdomain, we automatically attempt to secure the associated apex domain, but you need to make the DNS changes to use the `www` subdomain. For more information, see "[Managing a custom domain for your {% data variables.product.prodname_pages %} site](/articles/managing-a-custom-domain-for-your-github-pages-site)."
```
Comment (ethanpalm) [inline content/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages.md:31]:
```suggestion
We recommend always using a `www` subdomain, even if you also use an apex domain. When you create a new site with an apex domain, we automatically attempt to secure the `www` subdomain for use when serving your site's content, but you need to make the DNS changes to use the `www` subdomain. If you configure a `www` subdomain, we automatically attempt to secure the associated apex domain. For more information, see "[Managing a custom domain for your {% data variables.product.prodname_pages %} site](/articles/managing-a-custom-domain-for-your-github-pages-site)."
```

Applying the requested change from a subject matter expert who reviewed this ✨

**Review by ethanpalm — DISMISSED** [review]
Thanks for creating this PR, @alzafacon! I'm applying one small change that was requested by the Pages team and then I'll get this branch up to date and merged ⚡ 

Thank you for taking the time to improve the GitHub docs!

**Review by ethanpalm — APPROVED** [review]

**Pull request merged**